### PR TITLE
VZ-7662 initialize the Helm chart for the Verrazzano Cluster Operator

### PR DESCRIPTION
**Pull request by davisgomes**
Template helm chart for the Verrazzano Cluster Operator

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano-cluster-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+apiVersion: v2
```
Comment (desagar) [inline platform-operator/helm_config/charts/verrazzano-cluster-operator/Chart.yaml:3]: the app operator and platform operator have apiVersion as v1 in their charts. Should we use the same here?

File: platform-operator/helm_config/charts/verrazzano-cluster-operator/.helmignore [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022, Oracle and/or its affiliates.
```
Comment (desagar) [inline platform-operator/helm_config/charts/verrazzano-cluster-operator/.helmignore:1]: The helmignore files in the VPO and VAO charts are much smaller. Any reason why we need all these entries?

Comment (davisgomes) [reply platform-operator/helm_config/charts/verrazzano-cluster-operator/.helmignore:1]: This was the default from helm create. I don't think it hurts to include these.

File: platform-operator/helm_config/charts/verrazzano-cluster-operator/Chart.yaml (hunk 1 above)
Comment (davisgomes) [reply platform-operator/helm_config/charts/verrazzano-cluster-operator/Chart.yaml]: Yes we should. I'm not sure why the helm chart generated this way.

**Review by desagar — APPROVED** [review]

**Pull request merged**